MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -50,12 +50,22 @@ GeoMapper::~GeoMapper()
 
 void GeoMapper::mapOnDEM(const std::string &file_name)
 {
-	this->_raster = FileIO::AsciiRasterInterface::getRasterFromASCFile(file_name);
+	_raster = FileIO::AsciiRasterInterface::getRasterFromASCFile(file_name);
 	if (! _raster) {
 		ERR("GeoMapper::mapOnDEM(): failed to load %s", file_name.c_str());
 		return;
 	}
-	this->mapData();
+
+	std::vector<GeoLib::Point*> const* pnts(_geo_objects.getPointVec(_geo_name));
+	if (! pnts) {
+		ERR("Geometry \"%s\" does not exist.", _geo_name.c_str());
+		return;
+	}
+	if (GeoLib::isStation((*pnts)[0])) {
```

Comment:
The geometry management object `_geo_objects` (class `GEOObjects`) doesn't allow to insert an empty vector of points. Later, access to the vector of points is only possible via const methods
```vector<Point*> const* GEOObjects::getPointVec(...)```.

Of course, the user could cast away the const and remove points from the vector. This would have fatal consequences at some other locations in the GeoLib, too.

So the answer is no, but the user can use `GEOObjects` such that `pnts` could be empty.